lib/ansible/modules/commands/shell.py

```diff
@@ -69,7 +69,7 @@
 notes:
    -  If you want to execute a command securely and predictably, it may be
       better to use the M(command) module instead. Best practices when writing
-      playbooks will follow the trend of using M(command) unless M(shell) is
+      playbooks will follow the trend of using M(command) unless C(shell) is
```

Comment:
Looking at the actual produced documentation, there is another issue. Links are not very clear from the note styles used. Everything appears in white on blue, even the links. However, I would prefer if `M( )` would also cause the text to be in proportional font so it is clear it is a command (or at least something special, and not just a word.